# Chinese (China) translation of pybliographer.
# Copyright (C) 2004, 2010 Free Software Foundation, Inc.
# This file is distributed under the same license as the pybliographer package.
# Li Shaojie <storm-119@163.com>, 2004.
# lainme <lainme993@gmail.com>, 2010.
#   QAed by Yunqiang Su <wzssyqa@gmail.com>, 2010.
# YunQiang Su <wzssyqa@gmail.com>, 2011.
#
msgid ""
msgstr ""
"Project-Id-Version: pybliographer master\n"
"Report-Msgid-Bugs-To: http://bugzilla.gnome.org/enter_bug.cgi?"
"product=pybliographer&keywords=I18N+L10N&component=general\n"
"POT-Creation-Date: 2011-03-19 00:35+0000\n"
"PO-Revision-Date: 2011-04-10 16:21+0800\n"
"Last-Translator: YunQiang Su <wzssyqa@gmail.com>\n"
"Language-Team: Chinese (simplified) <i18n-zh@googlegroups.com>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: \n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: ../Pyblio/Autoload.py:78
#, python-format
msgid "warning: can't import %s: %s"
msgstr "警告：不能导入 %s：%s"

#: ../Pyblio/Config.py:52
#, python-format
msgid "value of `%s' should be of type %s"
msgstr "“%s”的值应该是 %s 类型"

#: ../Pyblio/Config.py:212
msgid "String"
msgstr "字符串"

#: ../Pyblio/Config.py:221
msgid "Boolean"
msgstr "布尔"

#: ../Pyblio/Config.py:240
msgid "Integer"
msgstr "整数"

#: ../Pyblio/Config.py:242
#, python-format
msgid "Integer under %d"
msgstr "小于 %d 的整数"

#: ../Pyblio/Config.py:244
#, python-format
msgid "Integer over %d"
msgstr "大于 %d 的整数"

#: ../Pyblio/Config.py:246
#, python-format
msgid "Integer between %d and %d"
msgstr "在 %d 和 %d 之间的整数"

#: ../Pyblio/Config.py:258
#, python-format
msgid "Element in `%s'"
msgstr "“%s”中的元素"

#: ../Pyblio/Config.py:278
#, python-format
msgid "Tuple (%s)"
msgstr "元组 (%s)"

#: ../Pyblio/Config.py:299
#, python-format
msgid "List (%s)"
msgstr "列表 (%s)"

#: ../Pyblio/Config.py:324
#, python-format
msgid "Dictionary (%s, %s)"
msgstr "字典 (%s，%s)"

#: ../Pyblio/Config.py:344
#, python-format
msgid "warning: could not restore setting %r to %r: %s"
msgstr "警告：无法将设置 %r 恢复到 %r: %s"

#: ../Pyblio/Fields.py:261
msgid "Illegal year value"
msgstr "非法的年份值"

#: ../Pyblio/Fields.py:265
msgid "Illegal month value"
msgstr "非法的月份值"

#: ../Pyblio/Fields.py:269
msgid "Illegal day value"
msgstr "非法的日期值"

#: ../Pyblio/Fields.py:510
#, python-format
msgid "in %s"
msgstr "在 %s"

#: ../Pyblio/Format/BibTeX.py:323 ../Pyblio/Format/BibTeX.py:371
#, python-format
msgid "key `%s' is malformed"
msgstr "键“%s”格式错误"

#: ../Pyblio/Format/BibTeX.py:340 ../Pyblio/Format/BibTeX.py:359
#, python-format
msgid "%s:%d: key `%s' already defined"
msgstr "%s：%d：键 %s 已定义"

#: ../Pyblio/GnomeUI/Config.py:147 ../Pyblio/GnomeUI/Fields.py:417
msgid ""
"Some changes require to restart Pybliographic\n"
"to be correctly taken into account"
msgstr "某些更改需要重新启动 Pybliographic 才能生效"

#: ../Pyblio/GnomeUI/Config.py:429
msgid "Add"
msgstr "添加"

#: ../Pyblio/GnomeUI/Config.py:432
msgid "Update"
msgstr "更新"

#: ../Pyblio/GnomeUI/Config.py:435 ../Pyblio/GnomeUI/Config.py:549
msgid "Remove"
msgstr "删除"

#: ../Pyblio/GnomeUI/Config.py:546
msgid "Set"
msgstr "设置"

#: ../Pyblio/GnomeUI/Config.py:561
msgid "Key:"
msgstr "键："

#: ../Pyblio/GnomeUI/Config.py:563
msgid "Value:"
msgstr "值："

#. id     stock            label         accel   tooltip   callback
#: ../Pyblio/GnomeUI/Document.py:145
msgid "_File"
msgstr "文件(_F)"

#: ../Pyblio/GnomeUI/Document.py:146 ../Pyblio/GnomeUI/Document.py:195
msgid "_Edit"
msgstr "编辑(_E)"

#: ../Pyblio/GnomeUI/Document.py:147
msgid "_View"
msgstr "查看(_V)"

#: ../Pyblio/GnomeUI/Document.py:148
msgid "_Cite"
msgstr "引用(_C)"

#: ../Pyblio/GnomeUI/Document.py:149
msgid "_Settings"
msgstr "设置(_S)"

#: ../Pyblio/GnomeUI/Document.py:150
msgid "_Help"
msgstr "帮助(_H)"

#: ../Pyblio/GnomeUI/Document.py:151
msgid "Recent documents"
msgstr "最近的文档"

#: ../Pyblio/GnomeUI/Document.py:154
msgid "Open a file"
msgstr "打开一个文件"

#: ../Pyblio/GnomeUI/Document.py:155
msgid "Open _Location"
msgstr "打开位置(_L)"

#: ../Pyblio/GnomeUI/Document.py:156
msgid "Save the current file"
msgstr "保存当前文件"

#: ../Pyblio/GnomeUI/Document.py:161
#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:8
msgid "Merge With..."
msgstr "合并对象..."

#: ../Pyblio/GnomeUI/Document.py:162
#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:7
msgid "Medline Query..."
msgstr "麦德林查询..."

#: ../Pyblio/GnomeUI/Document.py:170
msgid "Add a new entry"
msgstr "添加一个新项"

#: ../Pyblio/GnomeUI/Document.py:174
#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:12
msgid "S_ort..."
msgstr "排序(_O)..."

#: ../Pyblio/GnomeUI/Document.py:175
#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:2
msgid "Cite..."
msgstr "引用..."

#: ../Pyblio/GnomeUI/Document.py:175
msgid "Cite key(s)"
msgstr "引文键值"

#: ../Pyblio/GnomeUI/Document.py:176
#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:6
msgid "Format..."
msgstr "格式..."

#: ../Pyblio/GnomeUI/Document.py:178
#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:4
msgid "Fields..."
msgstr "字段..."

#: ../Pyblio/GnomeUI/Document.py:180
#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:5
msgid "Forget all changes"
msgstr "忽略所有改版"

#: ../Pyblio/GnomeUI/Document.py:183
msgid "_Resource"
msgstr "资源(_R)"

#: ../Pyblio/GnomeUI/Document.py:194
msgid "_About"
msgstr "关于(_A)"

#: ../Pyblio/GnomeUI/Document.py:245
msgid "Quick search"
msgstr "快速搜索"

#: ../Pyblio/GnomeUI/Document.py:319
msgid "Unnamed bibliographic database"
msgstr "未命名的书目数据库"

#: ../Pyblio/GnomeUI/Document.py:402
#, python-format
msgid ""
"can't open file `%s' for writing:\n"
"%s"
msgstr ""
"不能打开文件“%s”进行写入：\n"
"%s"

#: ../Pyblio/GnomeUI/Document.py:423
#, python-format
msgid ""
"Error while parsing `%s':\n"
"%s"
msgstr ""
"分析“%s”时出错：\n"
"%s"

#: ../Pyblio/GnomeUI/Document.py:439
msgid "New database"
msgstr "新建数据库"

#: ../Pyblio/GnomeUI/Document.py:447 ../Pyblio/GnomeUI/Document.py:451
msgid "[no entry]"
msgstr "[没有项]"

#: ../Pyblio/GnomeUI/Document.py:448
msgid "[1 entry]"
msgstr "[1 项]"

#: ../Pyblio/GnomeUI/Document.py:449
#, python-format
msgid "[%d entries]"
msgstr "[%d 项]"

#: ../Pyblio/GnomeUI/Document.py:452
#, python-format
msgid "[%d/1 entry]"
msgstr "[%d/1 项]"

#: ../Pyblio/GnomeUI/Document.py:453
#, python-format
msgid "[%d/%d entries]"
msgstr "[%d/%d 项]"

#: ../Pyblio/GnomeUI/Document.py:458
msgid "[modified]"
msgstr "[已修改]"

#: ../Pyblio/GnomeUI/Document.py:461
#, python-format
msgid "view limited to: %s"
msgstr "查看限制为：%s"

#: ../Pyblio/GnomeUI/Document.py:474
msgid ""
"The database has been modified.\n"
"Discard changes?"
msgstr ""
"数据库已经被修改。\n"
"放弃修改？"

#: ../Pyblio/GnomeUI/Document.py:478
msgid ""
"The database has been modified.\n"
"Save changes?"
msgstr ""
"数据库已经被修改。\n"
"保存修改？"

#: ../Pyblio/GnomeUI/Document.py:495
msgid ""
"The database has been externally modified.\n"
"Overwrite changes ?"
msgstr ""
"数据库有了外部修改。\n"
"覆盖修改？"

#. no result.
#: ../Pyblio/GnomeUI/Document.py:524
msgid "Your query returned no result"
msgstr "您的查询没有返回结果"

#. get a new file name
#: ../Pyblio/GnomeUI/Document.py:534
msgid "Merge file"
msgstr "合并文件"

#: ../Pyblio/GnomeUI/Document.py:547 ../Pyblio/GnomeUI/Document.py:634
msgid "Open error"
msgstr "打开错误"

#: ../Pyblio/GnomeUI/Document.py:571
msgid "Merge status"
msgstr "合并状态"

#. get a new file name
#: ../Pyblio/GnomeUI/Document.py:582
msgid "Open file"
msgstr "打开文件"

#: ../Pyblio/GnomeUI/Document.py:620
msgid ""
"An autosave file was found which is newer than the original file.\n"
"Do you want to restore it?"
msgstr ""
"发现了比原始文件新的自动保存文件。\n"
"您是否想要恢复？"

#: ../Pyblio/GnomeUI/Document.py:655 ../Pyblio/GnomeUI/Document.py:796
#: ../Pyblio/GnomeUI/Document.py:856
#, python-format
msgid ""
"Unable to remove autosave file `%s':\n"
"%s"
msgstr ""
"无法删除自动保存文件“%s”：\n"
"%s"

#: ../Pyblio/GnomeUI/Document.py:706
#, python-format
msgid ""
"Error during autosaving:\n"
"%s"
msgstr ""
"自动保存时出错：\n"
"%s"

#: ../Pyblio/GnomeUI/Document.py:734
#, python-format
msgid ""
"Unable to save `%s':\n"
"%s"
msgstr ""
"不能保存“%s”:\n"
"%s"

#: ../Pyblio/GnomeUI/Document.py:742
msgid ""
"An internal error occured during saving\n"
"Try to Save As..."
msgstr ""
"保存时发生内部错误\n"
"请尝试另存为..."

#: ../Pyblio/GnomeUI/Document.py:757
msgid "Save As..."
msgstr "另存为..."

#: ../Pyblio/GnomeUI/Document.py:763
#, python-format
msgid ""
"The file `%s' already exists.\n"
"Overwrite it ?"
msgstr ""
"文件“%s”已经存在。\n"
"覆盖它吗？"

#: ../Pyblio/GnomeUI/Document.py:770
#, python-format
msgid ""
"During opening:\n"
"%s"
msgstr ""
"打开时：\n"
"%s"

#: ../Pyblio/GnomeUI/Document.py:808
msgid "Reopen error"
msgstr "重新打开错误"

#: ../Pyblio/GnomeUI/Document.py:881
#, python-format
msgid ""
"An entry called `%s' already exists.\n"
"Rename and add it anyway ?"
msgstr ""
"名为“%s”的项已经存在。\n"
"无论如何重命名和添加它？"

#: ../Pyblio/GnomeUI/Document.py:918
msgid "Really remove all the entries ?"
msgstr "真的删除所有项吗？"

#: ../Pyblio/GnomeUI/Document.py:938
msgid "Create new entry"
msgstr "创建新项"

#: ../Pyblio/GnomeUI/Document.py:952
#, python-format
msgid "Really edit %d entries ?"
msgstr "真的编辑 %d 项"

#: ../Pyblio/GnomeUI/Document.py:998
#, python-format
msgid "Remove all the %d entries ?"
msgstr "删除所有 %d 项吗？"

#: ../Pyblio/GnomeUI/Document.py:1000
#, python-format
msgid "Remove entry `%s' ?"
msgstr "删除项“%s”吗？"

#: ../Pyblio/GnomeUI/Document.py:1019
msgid ""
"your search text must contain\n"
"latin-1 characters only"
msgstr "搜索词只能包含 latin-1 字符"

#: ../Pyblio/GnomeUI/Document.py:1098 ../Pyblio/GnomeUI/Document.py:1106
#, python-format
msgid ""
"Can't connect to LyX:\n"
"%s"
msgstr ""
"不能连接到 LyX：\n"
"%s"

#: ../Pyblio/GnomeUI/Document.py:1221
#, python-format
msgid ""
"Can't display documentation:\n"
"%s"
msgstr ""
"不能显示文档：\n"
"%s"

#: ../Pyblio/GnomeUI/Document.py:1229
msgid "This program is copyrighted under the GNU GPL"
msgstr "此软件版权遵照 GNU GPL "

#: ../Pyblio/GnomeUI/Document.py:1230
msgid "GNOME interface to the Pybliographer system."
msgstr "Gnome 到 Pybliographer 系统的接口。"

#: ../Pyblio/GnomeUI/Document.py:1241
msgid "GNOME Translation Team"
msgstr "Gnome 翻译小组"

#: ../Pyblio/GnomeUI/Document.py:1246
msgid "Pybliographer Home Page"
msgstr "Pybliographer 主页"

#: ../Pyblio/GnomeUI/Editor.py:35
msgid "Last Name, First Name"
msgstr "姓，名"

#: ../Pyblio/GnomeUI/Editor.py:36 ../Pyblio/GnomeUI/Editor.py:1292
#: ../Pyblio/GnomeUI/Fields.py:39 ../Pyblio/GnomeUI/glade/config1.glade.h:14
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:19
msgid "Text"
msgstr "文本"

#: ../Pyblio/GnomeUI/Editor.py:133 ../Pyblio/GnomeUI/Editor.py:1003
msgid "Error in native string parsing"
msgstr "本地字符语法分析发生错误"

#: ../Pyblio/GnomeUI/Editor.py:312
msgid "Day"
msgstr "日"

#: ../Pyblio/GnomeUI/Editor.py:321
msgid "Month"
msgstr "月"

#: ../Pyblio/GnomeUI/Editor.py:330
msgid "Year"
msgstr "年"

#: ../Pyblio/GnomeUI/Editor.py:355
msgid "Invalid day field in date"
msgstr "日期中无效的日字段"

#: ../Pyblio/GnomeUI/Editor.py:363
msgid "Invalid month field in date"
msgstr "日期中无效的月字段"

#: ../Pyblio/GnomeUI/Editor.py:371
msgid "Invalid year field in date"
msgstr "日期中无效的年字段"

#. A delete button
#: ../Pyblio/GnomeUI/Editor.py:419 ../Pyblio/GnomeUI/Search.py:86
msgid "Delete"
msgstr "删除"

#: ../Pyblio/GnomeUI/Editor.py:430 ../Pyblio/GnomeUI/Editor.py:443
#: ../Pyblio/GnomeUI/Editor.py:446
msgid "[ Drop an Entry here ]"
msgstr "[ 将项拖放到这里 ]"

#: ../Pyblio/GnomeUI/Editor.py:502
msgid "Browse..."
msgstr "浏览..."

#: ../Pyblio/GnomeUI/Editor.py:525
msgid "Select file"
msgstr "选择文件"

#: ../Pyblio/GnomeUI/Editor.py:549 ../Pyblio/GnomeUI/Fields.py:236
#: ../Pyblio/GnomeUI/glade/config1.glade.h:8
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:6
msgid "Entry type"
msgstr "输入类型"

#: ../Pyblio/GnomeUI/Editor.py:551
msgid "Key"
msgstr "键"

#: ../Pyblio/GnomeUI/Editor.py:600
msgid "Create Field"
msgstr "创建字段"

#. navigation buttons
#: ../Pyblio/GnomeUI/Editor.py:605
msgid "Back"
msgstr "后退"

#: ../Pyblio/GnomeUI/Editor.py:608
msgid "Next"
msgstr "前进"

#: ../Pyblio/GnomeUI/Editor.py:728 ../Pyblio/GnomeUI/Fields.py:451
msgid "Mandatory"
msgstr "必选"

#: ../Pyblio/GnomeUI/Editor.py:728 ../Pyblio/GnomeUI/Fields.py:452
msgid "Optional"
msgstr "可选"

#: ../Pyblio/GnomeUI/Editor.py:728 ../Pyblio/GnomeUI/Editor.py:1207
msgid "Notes"
msgstr "备忘"

#: ../Pyblio/GnomeUI/Editor.py:728
msgid "Extra"
msgstr "额外"

#: ../Pyblio/GnomeUI/Editor.py:909
msgid "Invalid key format"
msgstr "无效的键格式"

#: ../Pyblio/GnomeUI/Editor.py:917
#, python-format
msgid "Key `%s' already exists"
msgstr "键“%s”已经存在"

#: ../Pyblio/GnomeUI/Editor.py:934
#, python-format
msgid "The `%s' field contains a non Latin-1 symbol"
msgstr "“%s”字段包含一个非 Latin-1 符号"

#: ../Pyblio/GnomeUI/Editor.py:996
msgid "Your text contains non Latin-1 symbols"
msgstr "您的文本中包含有非 Latin-1 符号"

#: ../Pyblio/GnomeUI/Editor.py:1023
msgid "Edit entry"
msgstr "编辑项"

#: ../Pyblio/GnomeUI/Editor.py:1037 ../Pyblio/GnomeUI/Editor.py:1102
msgid "Native Editing"
msgstr "本地编辑"

#: ../Pyblio/GnomeUI/Editor.py:1125
msgid "Standard Editing"
msgstr "标准编辑"

#: ../Pyblio/GnomeUI/Editor.py:1229
msgid "mandatory"
msgstr "必选"

#: ../Pyblio/GnomeUI/Editor.py:1295
msgid "Enter text here"
msgstr "在此输入文本"

#: ../Pyblio/GnomeUI/Editor.py:1347
msgid "New Annotation Name"
msgstr "新批注名称"

#: ../Pyblio/GnomeUI/Editor.py:1353
msgid "Name of the new annotation:"
msgstr "新批注的名称："

#: ../Pyblio/GnomeUI/Entry.py:114
#, python-format
msgid ""
"Cannot open URL:\n"
"%s"
msgstr ""
"不能打开 URL：\n"
"%s"

#: ../Pyblio/GnomeUI/Fields.py:38
msgid "Authors"
msgstr "作者"

#: ../Pyblio/GnomeUI/Fields.py:40
msgid "Long Text"
msgstr "长文本"

#: ../Pyblio/GnomeUI/Fields.py:41 ../Pyblio/GnomeUI/glade/config1.glade.h:16
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:21
msgid "URL"
msgstr "URL"

#: ../Pyblio/GnomeUI/Fields.py:42
msgid "Reference"
msgstr "参考"

#: ../Pyblio/GnomeUI/Fields.py:43
msgid "Date"
msgstr "日期"

#: ../Pyblio/GnomeUI/Fields.py:124 ../Pyblio/GnomeUI/glade/config1.glade.h:11
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:10
msgid "Name"
msgstr "名称"

#: ../Pyblio/GnomeUI/Fields.py:127 ../Pyblio/GnomeUI/glade/config1.glade.h:15
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:20
msgid "Type"
msgstr "类型"

#: ../Pyblio/GnomeUI/Fields.py:300
msgid "Available"
msgstr "可用"

#: ../Pyblio/GnomeUI/Fields.py:310
msgid "Associated"
msgstr "关联"

#: ../Pyblio/GnomeUI/Fields.py:321
msgid "Please, select an entry type from previous page."
msgstr "请从前页选择一个项类型。"

#: ../Pyblio/GnomeUI/Fields.py:332
#, python-format
msgid "Fields associated with <b>%s</b> entry type"
msgstr "与<b>%s</b>关联的字段类型"

#: ../Pyblio/GnomeUI/FileSelector.py:39
msgid "File"
msgstr "文件"

#: ../Pyblio/GnomeUI/FileSelector.py:76
#: ../Pyblio/GnomeUI/glade/openurl.glade.h:2
msgid "Bibliography type:"
msgstr "参考文献类型："

#: ../Pyblio/GnomeUI/FileSelector.py:93 ../Pyblio/GnomeUI/OpenURL.py:61
msgid " - According to file suffix - "
msgstr " - 根据文件后缀 - "

#. The root of the search tree is the full database
#: ../Pyblio/GnomeUI/Search.py:71
msgid "Full database"
msgstr "整个数据库"

#: ../Pyblio/GnomeUI/Search.py:142
msgid "internal error during evaluation"
msgstr "鉴定时发生内部错误"

#: ../Pyblio/GnomeUI/Search.py:196
#, python-format
msgid ""
"while compiling %s\n"
"error: %s"
msgstr ""
"编译 %s 中\n"
"错误：%s"

#. These are additional search fields
#: ../Pyblio/GnomeUI/Sort.py:97
msgid "[Entry Type]"
msgstr "[项类型]"

#: ../Pyblio/GnomeUI/Sort.py:98
msgid "[Key Value]"
msgstr "[键值]"

#: ../Pyblio/GnomeUI/Utils.py:193
msgid "The following errors occured:\n"
msgstr "以下错误发生：\n"

#: ../Pyblio/GnomeUI/__init__.py:45
#, python-format
msgid "This is Pybliographic %s [Python %s, Gtk %s, PyGTK %s]"
msgstr "这是 Pybliographic %s [Python %s, Gtk %s, PyGTK %s] "

#: ../Pyblio/GnomeUI/glade/config1.glade.h:1
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:1
#: ../Pyblio/GnomeUI/glade/medline.glade.h:1
#: ../Pyblio/GnomeUI/glade/openurl.glade.h:1
#: ../Pyblio/GnomeUI/glade/search.glade.h:1
msgid "*"
msgstr "*"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:2
msgid "<b>Entry type</b>"
msgstr "<b>项类型</b>"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:3
msgid "A"
msgstr "A"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:4
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:2
msgid "Association"
msgstr "关联"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:5
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:3
msgid "Datum"
msgstr "资料"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:6
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:4
msgid "Editor"
msgstr "编辑"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:7
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:5
msgid "Entries"
msgstr "项"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:9
msgid "FIELDS DEFINITION"
msgstr "字段定义"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:10
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:8
msgid "Fields "
msgstr "字段"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:12
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:11
msgid "Name "
msgstr "名称"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:13
msgid "Pybliographer Configuration"
msgstr "图书书目管理器设置"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:17
msgid "standard fields"
msgstr "标准字段"

#: ../Pyblio/GnomeUI/glade/fields1.glade.h:7
msgid "Entry types and field names configuration"
msgstr "项类型和字段名称设置"

#: ../Pyblio/GnomeUI/glade/fields1.glade.h:9
msgid "Information for <b>Article</b> entry type"
msgstr "<b>文章</b>项类型信息"

#: ../Pyblio/GnomeUI/glade/fields1.glade.h:12
msgid ""
"Select an entry type and specify \n"
"its associated (mandatory and\n"
"optional) fields on the next page.\n"
"\n"
"\n"
"\n"
msgstr ""
"选择项类型并在下一页指定\n"
"与它关联(委托和可选)的字段。\n"
"\n"
"\n"
"\n"
"\n"

#: ../Pyblio/GnomeUI/glade/format.glade.h:1
msgid "Bibliography style:"
msgstr "书目样式："

#: ../Pyblio/GnomeUI/glade/format.glade.h:2
msgid "Format entries"
msgstr "格式项"

#: ../Pyblio/GnomeUI/glade/format.glade.h:3
msgid "Output file:"
msgstr "输出文件："

#: ../Pyblio/GnomeUI/glade/format.glade.h:4
msgid "Output format:"
msgstr "输出格式："

#: ../Pyblio/GnomeUI/glade/format.glade.h:5
msgid "Save Formatted as"
msgstr "保存格式为"

#: ../Pyblio/GnomeUI/glade/format.glade.h:6
msgid "Style Selection"
msgstr "样式选择"

#: ../Pyblio/GnomeUI/glade/medline.glade.h:2
msgid "Enter your Medline Query"
msgstr "输入您的 Medline 查询"

#: ../Pyblio/GnomeUI/glade/medline.glade.h:3
msgid "From:"
msgstr "来自："

#: ../Pyblio/GnomeUI/glade/medline.glade.h:4
msgid "Limited to:"
msgstr "限制为："

#: ../Pyblio/GnomeUI/glade/medline.glade.h:5
msgid ""
"Maximum number\n"
"of results:"
msgstr ""
"结果的最大\n"
"数目："

#: ../Pyblio/GnomeUI/glade/medline.glade.h:7
msgid "Medline Query"
msgstr "Medline 查询"

#: ../Pyblio/GnomeUI/glade/medline.glade.h:8
msgid ""
"Only items \n"
"ahead of print"
msgstr ""
"仅打印在\n"
"前的项"

#: ../Pyblio/GnomeUI/glade/medline.glade.h:10
msgid ""
"Only items \n"
"with abstracts"
msgstr ""
"仅为有摘要\n"
"的项"

#: ../Pyblio/GnomeUI/glade/medline.glade.h:12
msgid ""
"Search \n"
"PubMed for:"
msgstr ""
"搜索\n"
"PubMed:"

#: ../Pyblio/GnomeUI/glade/medline.glade.h:14
msgid ""
"Starting listing at\n"
"result number:"
msgstr ""
"在结果数目\n"
"开始列出："

#: ../Pyblio/GnomeUI/glade/medline.glade.h:16
msgid "To:"
msgstr "到："

#: ../Pyblio/GnomeUI/glade/medline.glade.h:17
msgid ""
"Use the format YYYY/MM/DD\n"
"Month and day are optional."
msgstr ""
"使用格式 YYYY/MM/DD\n"
"月和日是可选的。"

#: ../Pyblio/GnomeUI/glade/openurl.glade.h:3
msgid "Enter the location (URI) of the file you would like to open:"
msgstr "输入您想要打开的文件的位置(URI)："

#: ../Pyblio/GnomeUI/glade/openurl.glade.h:4
msgid "Open Location"
msgstr "打开位置"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:1
msgid "Cite"
msgstr "引用"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:3
msgid "Contents"
msgstr "内容"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:9
msgid "Open File"
msgstr "打开文件"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:10
msgid "Previous Documents"
msgstr "之前文档"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:11
msgid "Pybliographer"
msgstr "图书目录管理器"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:13
msgid "Save File"
msgstr "保存文件"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:14
msgid "Settings"
msgstr "设置"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:15
msgid "_Add..."
msgstr "添加(_A)..."

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:16
msgid "_Delete..."
msgstr "删除(_D)..."

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:17
msgid "_Edit..."
msgstr "编辑(_E)..."

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:18
msgid "_New"
msgstr "新建(_N)"

#: ../Pyblio/GnomeUI/glade/search.glade.h:2
msgid "Close"
msgstr "关闭"

#: ../Pyblio/GnomeUI/glade/search.glade.h:3
msgid "Expert Search"
msgstr "专业搜索"

#: ../Pyblio/GnomeUI/glade/search.glade.h:4
msgid "Field"
msgstr "字段"

#: ../Pyblio/GnomeUI/glade/search.glade.h:5
msgid "Pattern"
msgstr "模式"

#: ../Pyblio/GnomeUI/glade/search.glade.h:6
msgid "Query"
msgstr "查询"

#: ../Pyblio/GnomeUI/glade/search.glade.h:7
msgid "Search"
msgstr "搜索"

#: ../Pyblio/GnomeUI/glade/search.glade.h:8
msgid "Simple Search"
msgstr "简单搜索"

#: ../Pyblio/GnomeUI/glade/sort.glade.h:1
msgid "Select sort criterions"
msgstr "选择排序标准"

#: ../Pyblio/GnomeUI/glade/sort.glade.h:2
msgid "Set as default"
msgstr "设为默认"

#: ../Pyblio/LyX.py:37
#, python-format
msgid "no input pipe `%s'"
msgstr "没有输入管道“%s”"

#: ../Pyblio/LyX.py:43
#, python-format
msgid "no output pipe `%s'"
msgstr "没有输出管道“%s”"

#: ../Pyblio/Open.py:96
#, python-format
msgid "method `%s' provides no opener"
msgstr "方法“%s”未提供开端"

#: ../Pyblio/Open.py:104 ../Pyblio/Open.py:151
#, python-format
msgid "File `%s' does not exist"
msgstr "文件“%s”不存在"

#: ../Pyblio/Open.py:115 ../Pyblio/Open.py:162
#, python-format
msgid "don't know how to open `%s'"
msgstr "不知道如何打开“%s”"

#: ../Pyblio/Open.py:143
#, python-format
msgid "method `%s' provides no iterator"
msgstr "方法“%s”没有提供迭代器"

#: ../Pyblio/Resource.py:77
#, python-format
msgid ""
"Warning: This URL is marked as Invalid or Approximate: %s\n"
"Continue?"
msgstr ""
"警告：此 URL 被标记为无效或不确切的：%s\n"
"继续吗？"

#: ../Pyblio/Resource.py:81
msgid "Determining Mime Type ... "
msgstr "正在确定 MIME 类型..."

#: ../Pyblio/Resource.py:86
#, python-format
msgid "Cannot determine mime type for item %s "
msgstr "无法确定项目 %s 的 MIME 类型"

#: ../Pyblio/Resource.py:87
#, python-format
msgid ""
"URL in question is: %s\n"
"You should check the url or path given for errors.\n"
"Details: %s"
msgstr ""
"有问题的 URL 是：%s\n"
"您应该检查 URL 或给定路径中的错误。\n"
"细节：%s"

#: ../Pyblio/Resource.py:98
msgid "Accessing resource ..."
msgstr "正在访问资源..."

#: ../Pyblio/Resource.py:116
#, python-format
msgid "IOError for item %s: cannot uncompress resource."
msgstr "项目 %s 的 IO 错误：无法解压缩资源。"

#: ../Pyblio/Resource.py:117
#, python-format
msgid ""
"URL: %s\n"
"Details: %s"
msgstr ""
"URL：%s\n"
"细节：%s"

#: ../Pyblio/Resource.py:137
msgid "Starting application ..."
msgstr "正在启动应用程序..."

#: ../Pyblio/Resource.py:140
msgid "No application to view resource"
msgstr "没有查看资源的应用程序"

#: ../Pyblio/Resource.py:141
#, python-format
msgid ""
"For mime type %s, no entry found in \n"
"configuration option resource/viewers.\n"
"Please consider adding one.\n"
"URL: %s"
msgstr ""
"对于 MIME 类型 %s，在配置选项\n"
"资源/查看器中没有找到项。\n"
"请考虑添加一个。\n"
"URL：%s"

#: ../Pyblio/Style/Parser.py:261
#, python-format
msgid "missing '%s' attribute"
msgstr "丢失“%s”属性"

#: ../Pyblio/Style/Parser.py:271
#, python-format
msgid "invalid opening tag: %s"
msgstr "无效打开标记：%s"

#: ../Pyblio/Style/Parser.py:281
#, python-format
msgid "invalid closing tag: %s"
msgstr "无效关闭标记：%s"

#: ../pybliographer.py:43
#, python-format
msgid "This is %s, version %s"
msgstr "这是 %s，版本 %s"

#: ../pybliographer.py:47 ../pybliographer.py:54
msgid "This is free software with ABSOLUTELY NO WARRANTY."
msgstr "此为自由软件，没有绝对的保证书"

#: ../pybliographer.py:48
msgid "For details, type `warranty'."
msgstr "获得细节，请输入“warranty”。"

#: ../pybliographer.py:111 ../pybliographer.py:163
#, python-format
msgid "%s: error: can't open file `%s'"
msgstr "%s：错误：不能打开文件“%s”"

#: ../pybliographer.py:124
#, python-format
msgid "For help, run %s and type `help' at the prompt"
msgstr "需要帮助时，运行 %s 并且在提示符处键入“help”"

#: ../pybliographer.py:133
msgid ""
"Useful commands:\n"
"\thelp     to get some help\n"
"\tquit     to quit\n"
msgstr ""
"有用的命令：\n"
"\thelp     获得帮助\n"
"\tquit     退出\n"

#. save history
#: ../pybliographic.desktop.in.h:1
msgid "Bibliography Manager"
msgstr "书目管理器"

#: ../pybliographic.desktop.in.h:2
msgid "Manage bibliographic databases"
msgstr "管理书目数据库"

#: ../pybliographic.desktop.in.h:3
msgid "Pybliographic Bibliography Manager"
msgstr "图书书目管理器"

#: ../scripts/pybcheck.py:31
msgid "usage: pybcheck <file | directory>..."
msgstr "用法：pybcheck <文件 | 目录>..."

#: ../scripts/pybcheck.py:60
#, python-format
msgid "file `%s' is ok [%d entries]"
msgstr "文件“%s”正常[%d 项]"

#: ../scripts/pybcompact.py:30
msgid "usage: pybcompact <latexfile> <bibtexfiles...>"
msgstr "用法：pybcompact <latexfile> <bibtexfiles...>"

#: ../scripts/pybcompact.py:34
#, python-format
msgid "pybcompact: error: %s\n"
msgstr "pybcompact：错误：%s\n"

#: ../scripts/pybcompact.py:99
msgid "no entry"
msgstr "没有项"

#: ../scripts/pybcompact.py:162 ../scripts/pybtext.py:240
#, python-format
msgid "can't find the following entries: %s"
msgstr "找不到以下项：%s"

#: ../scripts/pybconvert.py:29
msgid "usage: pybconvert <source>..<target> <input> [output]"
msgstr "用法：pybconvert <source>..<target> <input> [output]"

#: ../scripts/pybconvert.py:38
msgid "pybconvert: error: bad conversion format"
msgstr "pybconvert：错误：错误的转换格式"

#: ../scripts/pybformat.py:34
#, fuzzy
#| msgid ""
#| "usage: pybformat [options] <database...>\n"
#| "\n"
#| "    options:\n"
#| "      -o file, --output=file\t\tspecify an output filename\n"
#| "      -s style, --style=style\t\tspecify a bibliography style\n"
#| "      -f format, --format=format\tspecify an output format\n"
#| "      -H header, --header=header\tdefines a header file\n"
#| "      -F footer, --footer=footer\tdefines a footer file\n"
#| "      -l 'output', --list='output'\tlists the available output formats\n"
#| "      -h, --help\t\t\tshow this help message\n"
#| "      -v, --version\t\t\tshow this help message\n"
#| "    "
msgid ""
"usage: pybformat [options] <database...>\n"
"\n"
"    options:\n"
"      -o file, --output=file\t\tspecify an output filename\n"
"      -s style, --style=style\t\tspecify a bibliography style\n"
"      -f format, --format=format\tspecify an output format\n"
"      -H header, --header=header\tdefines a header file\n"
"      -F footer, --footer=footer\tdefines a footer file\n"
"      -l 'output', --list='output'\tlists the available output formats\n"
"      -h, --help\t\t\tshow this help message\n"
"    "
msgstr ""
"用法：pybformat [选项] <数据库...>\n"
"\n"
"    选项：\n"
"      -o file, --output=file\t\t指定一个输出文件名\n"
"      -s style, --style=style\t\t指定一个文献目录样式\n"
"      -f format, --format=format\t指定一个输出格式\n"
"      -H header, --header=header\t定义一个页眉文件\n"
"      -F footer, --footer=footer\t定义一个页脚文件\n"
"      -l 'output', --list='output'\t列出可用的输出格式\n"
"      -h, --help\t\t\t显示这个帮助信息\n"
"      -v, --version\t\t\t显示版本信息\n"
"    "

#: ../scripts/pybformat.py:48
#, python-format
msgid "pybformat: error: %s\n"
msgstr "pybformat：错误：%s\n"

#: ../scripts/pybformat.py:54
#, python-format
msgid "pybformat: warning: %s\n"
msgstr "pybformat：警告：%s\n"

#: ../scripts/pybformat.py:93 ../scripts/pybtext.py:252
#: ../scripts/pybtext.py:278 ../scripts/pybtext.py:321
#, python-format
msgid "can't open `%s': %s"
msgstr "不能打开“%s”：%s"

#: ../scripts/pybformat.py:100
#, python-format
msgid "unknown list `%s'"
msgstr "未知列表“%s”"

#: ../scripts/pybformat.py:103
#, python-format
msgid "pybformat: available values for `%s':"
msgstr "pybformat：“%s”的可用值："

#: ../scripts/pybformat.py:107
#, python-format
msgid "empty value list `%s'"
msgstr "空的值列表“%s”"

#: ../scripts/pybformat.py:129
msgid "too few arguments"
msgstr "参数太少"

#: ../scripts/pybformat.py:136 ../scripts/pybtext.py:117
#, python-format
msgid "unknown output format `%s'"
msgstr "未知的输出管道格式“%s”"

#: ../scripts/pybformat.py:149 ../scripts/pybtex.py:86
#: ../scripts/pybtext.py:94
#, python-format
msgid "can't find style `%s'"
msgstr "找不到样式“%s”"

#: ../scripts/pybformat.py:152
#, python-format
msgid "pybformat: using style `%s', format `%s'\n"
msgstr "pybformat：使用样式“%s”，格式“%s”\n"

#: ../scripts/pybformat.py:167
#, python-format
msgid "can't open header `%s': %s"
msgstr "不能打开页眉“%s”：%s"

#: ../scripts/pybformat.py:175
#, python-format
msgid "can't open database: %s"
msgstr "无法打开数据库：%s"

#: ../scripts/pybformat.py:190
#, python-format
msgid "can't open footer `%s': %s"
msgstr "不能打开页脚“%s”：%s"

#: ../scripts/pybtex.py:34
msgid "usage: pybtex <latexfile> [bibtexfiles...]"
msgstr "用法：pybtex <latexfile> [bibtexfiles...]"

#: ../scripts/pybtex.py:38
#, python-format
msgid "pybtex: error: %s\n"
msgstr "pybtex：错误：%s\n"

#. warn the user that some entries were not found
#: ../scripts/pybtex.py:59
msgid "pybtex: warning: the following keys were not resolved"
msgstr "pybtex：警告：以下键未解决"

#. If the LaTeX document declares no style...
#: ../scripts/pybtex.py:64
msgid "no style defined"
msgstr "没有定义样式"

#: ../scripts/pybtext.py:41
msgid "usage: pybtext [-o outputfile] [-s style] <textfile> <bibfiles...>"
msgstr "用法：pybtext [-o 输出文件] [-s 样式] <文本文件> <bibfiles...>"

#: ../scripts/pybtext.py:45
#, python-format
msgid "pybtext: error: %s\n"
msgstr "pybtext：错误：%s\n"

#: ../scripts/pybtext.py:50
#, python-format
msgid "pybtext: warning: %s\n"
msgstr "pybtext：警告：%s\n"

#: ../scripts/pybtext.py:124
#, python-format
msgid "File already exists: `%s'"
msgstr "文件已经存在：“%s”"

#: ../scripts/pybtext.py:127
#, python-format
msgid "A file with the same name already exists: `%s'"
msgstr "已经存在同名文件：“%s”"

#: ../scripts/pybtext.py:178
msgid "no citation found"
msgstr "没有找到引用"

#: ../scripts/pybtext.py:193
#, python-format
msgid "pybtext: using style `%s', format `%s'\n"
msgstr "pybtext：使用样式“%s”，格式“%s”\n"

#: ../scripts/pybtext.py:316
#, python-format
msgid "can't create `%s'"
msgstr "无法创建“%s”"

#: ../scripts/pybtext.py:342
#, python-format
msgid "can't remove `%s'"
msgstr "无法删除“%s”"

#: ../scripts/pybtext.py:344
msgid "Done"
msgstr "完成"

#~ msgid "Failure for item %s"
#~ msgstr "项目 %s 失败"

#~ msgid ""
#~ "No viewer found in Config resource/viewers\n"
#~ "URL: %s"
#~ msgstr ""
#~ "配置资源/查看器中未发现查看器\n"
#~ "URL：%s"
